### ISSUE TITLE
Inaccurate informations in "Other projects" section in README.md

### ISSUE THREAD

**Issue by callb4ck** [ahayzen/folderbox#16]
The description for distrobox appears to be wrong because of the following reasons:

1) Declarative appears as "No" when distrobox actually supports a declarative manifest format ([distrobox assemble](https://github.com/89luca89/distrobox/blob/main/docs/usage/distrobox-assemble.md)).

2) There are multiple ways to run commands directly on the host system with no need for superuser priviledges, so might as well change sandbox from "Weak" to "None".

3) Default folder access appears as "Home", when actually distrobox by default mounts the host root in `/run/host/` and it's accessible directly from the container with again no need for superuser priviledges.

**Comment by ahayzen** [comment]
Good point, this was made before `distrobox assemble` existed, I've been planning to update the readme as the vscode part i don't use anymore so is untested etc. So I'll try and refactor this over the next few weeks after the holidays :-)
